### ISSUE TITLE
[English Town] - Debating Vocabulary

### ISSUE THREAD

**Issue by viphat** [viphat/til#229]
### Starting an opinion:

- In our opinion...
- We (don't) think that...
- The way we see it...
- If you want our honest opinion...
- According to me... According to the other side/our opponents...
- As far as I'm concerned...
- Our position is the following...

### Sequencing

- Firstly... secondly,... our third point is that...
- The first good reason to... is that...; Next; What more; Moreover...
- To begin, we think that...; in addiction, you have to know that...; Last but not least...
- The first point I would like to raise is this...
- Here's the main point I want to raise...
- I'd like to deal with two points here. The first is...

### I'm listening to the other side

- I see you point, but I think...
- Yes, I understand, but my opinion is that...
- I beg to differ.
- That's all very interesting, but the problem is that...
- I'm afraid I can't quite agree with your point.
- I think I've got your point, now let me respond to it.
- We can see what you're saying. Here's my reply...

### Disagreeing

- Excuse me, but that's not quite correct.
- Sorry, I just have to disagree with your point.
- Let me just respond to that, please.
- I'd like to take issue with what you just said.
- We said that... but the other side has not replied to our point.
- I'd like to focus on two points that the other side has failed to address.
- There are two issues our opponents have failed to dispute, namely...
- We pointed out that...
- Our opponents have claimed that...

### Finishing

- To recap the main points...
- Let's sum up where we stand in this debate...
- In summary, we want to point out that...
